sql/moz-fx-data-shared-prod/mozilla_vpn_derived/funnel_product_page_to_subscribed_v1/query.sql

```diff
@@ -108,7 +108,7 @@ flows AS (
     flow_id
   HAVING
     -- NOTE: flows near date boundaries may not meet this condition for all dates
-    "guardian-vpn" IN UNNEST(ARRAY_AGG(DISTINCT service))
+    LOGICAL_OR("guardian-vpn" = service)
```

Comment:
more efficient calculation with the same result

Reply:
Good call.  I think it'd read more naturally with the comparison value on the right though.
```suggestion
    LOGICAL_OR(service = "guardian-vpn")
```